rest-api/9-complete-with-all-defence-layers/DataContracts/ProductDataContract.cs

```diff
@@ -2,12 +2,9 @@
 
 namespace Defence.In.Depth.DataContracts;
 
-[DataContract]
-public record ProductDataContract
+public interface IDataContract
 {
-    [DataMember]
-    public string? Id { get; init; }
-        
-    [DataMember]
-    public string? Name { get; init; }
-}
\ No newline at end of file
+}
+
+[DataContract]
```

Comment:
There is no need for the `DataContract` attribution if we do not also add `DataMember`.  I think that `DataContract` should be removed.

Reply:
ok, will fix this!